connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/registry/InboundConnectorRegistry.java

```diff
@@ -69,7 +75,9 @@ public void registerWebhookConnector(InboundConnectorProperties properties) {
     WebhookConnectorProperties webhookConnectorProperties =
         new WebhookConnectorProperties(properties);
 
-    String bpmnId = webhookConnectorProperties.getBpmnProcessId();
+    ProcessCorrelationPoint correlationPoint = properties.getCorrelationPoint();
+    String bpmnId = correlationPoint.getBpmnProcessId();
```

Comment:
OK - I think I get the idea why you want to have the processId in the CorrelationPoint. 

I still find this a little bit confusing. As an alternative we could add the relevant properties also to the InboundConnectorProperties directly? Then we would separate the two pieces of information: Process definition defining the inbound connector - and the correlation point itself (completely opaque here). 

But also just my gut feeling - feel free to ignore :-)